bin/lint-back-diff

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+DIFF_FILES="$(git diff --name-only master HEAD | grep -e .py$ | grep -v migrations | tr "\n" " ")" || true
```

Comment:
I suggest you the following that allows to run back-end linters even on un-committed changes (not only committed changes compared to `master`):

```suggestion
DIFF_FILES="$(git diff --name-only master | grep -e .py$ | grep -v migrations | tr "\n" " ")" || true
```